CAP/gap/DerivedMethods.gi

```diff
@@ -654,6 +654,16 @@ AddDerivationToCAP( IsTerminal,
 end : Description := "IsTerminal using IsZeroForObjects",
       CategoryFilter := IsAdditiveCategory ); #Ab-Category?
 
+##
+AddDerivationToCAP( IsTerminal,
+                  
+  function( object )
+    
+    return IsIsomorphism( UniversalMorphismIntoTerminalObject( object ) );
+    
+end : Description := "IsTerminal using IsIsomorphism( UniversalMorphismIntoTerminalObject )",
+      CategoryFilter := IsCapCategory );
```

Comment:
Setting the `CategoryFilter` for a derivation that applies to arbitrary categories
is okay, but not strictly needed, since it is set as a default by CAP.

Reply:
Shall I remove it in a new commit?

Reply:
That would be nice since after the removal,
your derivations would match the other derivations
that do not depend on extra categorical properties.

Reply:
Done.